src/helpers/helpers.extras.ts

```diff
@@ -1,7 +1,9 @@
+import {type ChartMeta, type PointElement} from '../../types';
+
 import {_limitValue} from './helpers.math';
 import {_lookupByKey} from './helpers.collection';
 
-export function fontString(pixelSize, fontStyle, fontFamily) {
+export function fontString(pixelSize: number, fontStyle: string, fontFamily: string) {
```

Comment:
Do we want the `fontStyle` types as the different types like bold, italic etc?

Reply:
We could. Not sure how complicated that might be given all the combinations of styles that could be supplied